yew-ui/static/style.css

```diff
@@ -46,6 +46,15 @@ canvas {
   position: relative;
 }
 
+.grid-item-pinned {
```

Comment:
You could use tailwind in the rust files instead of this but I'm fine with it either way

Reply:
I agree, I am still learning tailwind, I'll come back and simplify this